pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -45,4 +45,9 @@
 .. _specifications: https://packaging.python.org/en/latest/specifications/
 .. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
 """,
+    "typing": """\
+Many recent PEPs propose changes to Python's static type system
+or otherwise relate to type annotationsqq.
```

Comment:
```suggestion
or otherwise relate to type annotations.
```

Cat walked on your keyboard? :joy_cat:

Reply:
I have only myself to blame :D